pages/gatsby.yml

```
@@ -39,7 +39,7 @@ jobs:
             echo "::set-output name=manager::yarn"
             echo "::set-output name=command::install"
             exit 0
-          elif [ -f "${{ github.workspace }}/package-lock.json" ]; then
+          else [ -f "${{ github.workspace }}/package.json" ]; then
```

Was originally going to get rid of the condition so switched to `else` but then changed my mind without fixing the syntax:

```suggestion
          elif [ -f "${{ github.workspace }}/package.json" ]; then
```